File: TransitoAplicacionWeb/Transito/Models/Conductor.cs
using System;
using System.Collections.Generic;

namespace Transito.Models
{
    public partial class Conductor
    {
        public Conductor()
        {
            ConductorBitacoraAcceso = new HashSet<ConductorBitacoraAcceso>();
            Reporte = new HashSet<Reporte>();
            Vehiculo = new HashSet<Vehiculo>();
        }

        public int Idconductor { get; set; }
        public string Nombre { get; set; }
        public string ApellidoPaterno { get; set; }
        public string ApellidoMaterno { get; set; }
        public DateTime FechaNacimiento { get; set; }
        public string NumeroLicencia { get; set; }
        public string Telefono { get; set; }
        public string Password { get; set; }

        public ICollection<ConductorBitacoraAcceso> ConductorBitacoraAcceso { get; set; }
        public ICollection<Reporte> Reporte { get; set; }
        public ICollection<Vehiculo> Vehiculo { get; set; }
    }
}

File: TransitoAplicacionMovil/app/src/main/java/mx/uv/lunix/transito/ws/pojos/Conductor.java
package mx.uv.lunix.transito.ws.pojos;

public class Conductor {

    private Integer IdConductor;
    private String Nombre;
    private String ApellidoPaterno;
    private String ApellidoMaterno;
    private String FechaNacimiento;
    private String NumeroLicencia;
    private String Telefono;
    private String Password;

    public Conductor(Integer idConductor, String nombre, String apellidoPaterno, String apellidoMaterno, String fechaNacimiento, String numeroLicencia, String telefono, String password) {
        IdConductor = idConductor;
        Nombre = nombre;
        ApellidoPaterno = apellidoPaterno;
        ApellidoMaterno = apellidoMaterno;
        FechaNacimiento = fechaNacimiento;
        NumeroLicencia = numeroLicencia;
        Telefono = telefono;
        Password = password;
    }

    public Integer getIdConductor() {
        return IdConductor;
    }

    public void setIdConductor(Integer idConductor) {
        IdConductor = idConductor;
    }

    public String getNombre() {
        return Nombre;
    }

    public void setNombre(String nombre) {
        Nombre = nombre;
    }

    public String getApellidoPaterno() {
        return ApellidoPaterno;
    }

    public void setApellidoPaterno(String apellidoPaterno) {
        ApellidoPaterno = apellidoPaterno;
    }

    public String getApellidoMaterno() {
        return ApellidoMaterno;
    }

    public void setApellidoMaterno(String apellidoMaterno) {
        ApellidoMaterno = apellidoMaterno;
    }

    public String getFechaNacimiento() {
        return FechaNacimiento;
    }

    public void setFechaNacimiento(String fechaNacimiento) {
        FechaNacimiento = fechaNacimiento;
    }

    public String getNumeroLicencia() {
        return NumeroLicencia;
    }

    public void setNumeroLicencia(String numeroLicencia) {
        NumeroLicencia = numeroLicencia;
    }

    public String getTelefono() {
        return Telefono;
    }

    public void setTelefono(String telefono) {
        Telefono = telefono;
    }

    public String getPassword() {
        return Password;
    }

    public void setPassword(String password) {
        Password = password;
    }
}
